src/components/learner-credit-management/BudgetStatusSubtitle.jsx

```diff
@@ -8,19 +9,15 @@ import { GroupAdd, Groups, ManageAccounts } from '@openedx/paragon/icons';
 
 import { useIntl } from '@edx/frontend-platform/i18n';
 import {
-  formatDate,
-  useEnterpriseCustomer,
-  useEnterpriseGroup,
-  isLmsBudget,
+  formatDate, useEnterpriseCustomer, useEnterpriseGroup, isLmsBudget,
 } from './data';
 
 const BudgetStatusSubtitle = ({
   badgeVariant, status, isAssignable, term, date, policy, enterpriseUUID, isRetired,
 }) => {
+  const customGroup = !isEmpty(policy?.groupAssociations);
```

Comment:
1. Can we use `enterpriseGroup` from the next line for this check?
2. I think this var will also be true if there is a global group associated with the policy, so we may need to add another check here for the `appliesToAllContexts` value